front-end/src/styles/App.css

```diff
@@ -68,3 +68,31 @@ button {
 button:hover {
   background-color: rgb(238, 48, 86);
 }
+
+select {
+  width: 200px;
+  padding: 5px;
+  font-size: 12px;
+}
```

Comment:
Why is there select styling here?

Reply:
I have no idea, trying to find where it was added in and cant find it. Either way I removed it and nothing changed